Switch back to the default checkout action

bhack opened the pull request:
/cc @malfet 

> The following actions use a deprecated Node.js version and will be forced to run on node20: malfet/checkout@silent-checkout, actions/setup-python@v4. For more info: https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/

https://github.com/pytorch/pytorch/actions/runs/11465627688
https://github.com/pytorch/pytorch/actions/runs/11465627688

bhack committed:
Switch back to the default action

bhack committed:
Fix typo

malfet reviewed (dismissed):
Glad to see they've finally implemented it

bhack commented:
> Glad to see they've finally implemented it

It was https://github.com/actions/checkout/pull/1067

But this other one is still not merged so I don't know if it is an issue for us:
https://github.com/actions/checkout/pull/1454

malfet commented:
> > Glad to see they've finally implemented it
> 
> It was [actions/checkout#1067](https://github.com/actions/checkout/pull/1067)
> 
> But this other one is still not merged so I don't know if it is an issue for us: [actions/checkout#1454](https://github.com/actions/checkout/pull/1454)

I can update my copy of action to use node20, because looks like checkout step is still much noisier than it used to be. On the other hand, I'm not sure if anyone still cares. 
@ezyang is this still an issue to you?

bhack commented:
> I can update my copy of action to use node20, because looks like checkout step is still much noisier than it used to be

Can you point me to the log?

bhack committed:
Complete action conversion coverage

bhack commented:
I've completed the coverage,  I don't know if in few cases `quiet-checkout` was missing.

bhack commented on .github/workflows/create_release.yml:
```diff
@@ -35,7 +35,7 @@ jobs:
     steps:
-      - uses: malfet/checkout@silent-checkout
+      - uses: actions/checkout@v4
         with:
```
E.g. if `quiet-checkout` was missing here in `main` we could add `show-progress: false` here and in few other places

ezyang commented:
Non hidden progress is annoying but if we need to upgrade I can deal

bhack commented:
@ezyang Where are you going to see the new actions logs? I still see the old one 
![immagine](https://github.com/user-attachments/assets/bca8b576-08b2-4ff8-82b9-71eb721d9fc4)

bhack commented:
I think that it is still cached and we are using the old one in the CI to test this.

bhack commented:
Can we do something to run these new actions in the CI?

bhack commented:
Can we try tun run the actions in this PR in the CI? As the CI seems to have ran the old cached one in `main`. 

See the action log:
https://github.com/pytorch/pytorch/actions/runs/11488525801/job/31975420465?pr=138739#step:2:26

bhack committed:
Trigger rebuild

bhack commented:
I have made a dummy commit. I have also closed and reopened the PR but I still see the CI executing `malfet/checkout@silent-checkout`

bhack committed:
Merge branch 'main' into pr/bhack/138739-4

bhack committed:
Switch to default action updated windows action

bhack commented:
I've re-based this again as in the meaintime you have merged other PR in conflicts.

bhack committed:
Merge branch 'main' into pr/bhack/138739-4

bhack commented:
I've rebased this gain. I hope we could test this in the CI

bhack commented:
Can we re-run the CI? These failures are unrelated.

bhack committed:
Merge branch 'main' into pr/bhack/138739-4

bhack committed:
Merge branch 'main' into pr/bhack/138739-4

bhack committed:
Trigger Build

bhack commented:
What is the status of this?

bhack committed:
Merge branch 'main' into patch-8

bhack committed:
Merge branch 'main' into patch-8

bhack commented:
Resolved merge conflicts again

bhack commented:
There is no interest I am going to close it.

malfet commented:
@bhack I believe I've approved it a while back, not sure what happened to it

bhack committed:
Merge branch 'main' into patch-8

bhack commented:
I've rebased this for two month with 11 messages/requests without reply. It is quite hard to contribute, I have another Docker PR parked and rebased by months.

malfet commented:
@pytorchbot merge -f "Lint is green and otherwise it's a no-op"

malfet commented:
<img width="820" alt="image" src="https://github.com/user-attachments/assets/fd1d42fb-e7dd-4cdb-bd32-c15807a901ed" />
Anyone knows how to interpret this message? It looks like I've approved it a while back, but than somebody (or something) dismissed my review?

bhack commented:
Diverged again

bhack commented:
What do you want to do with this?

bhack commented:
@malfet it is constantly conflicting merge after meege. Do we want to close it?

bhack commented:
Too much time for a review since october 2024. I am closing this.
